tests/phpunit/tests/user/author.php

```diff
@@ -144,4 +145,42 @@ public function test_get_the_author_posts_link_with_permalinks() {
 		unset( $GLOBALS['authordata'] );
 	}
 
+	/**
+	 * @ticket 51859
+	 */
+	public function test_get_author_url_link() {
+		$author_url          = get_the_author_meta( 'url' );
+		$author_display_name = get_the_author();
+
+		$author_url_link_html = sprintf(
+			'<a href="%1$s" title="%2$s" rel="author external">%3$s</a>',
```

Comment:
I'm not sure if it's helpful to test the full markup here, as this pretty much duplicates the entire function. Looking at the two tests for `get_the_author_posts_link()`, I think just testing that the output contains both the author URL and the display name might be enough here.

Reply:
@SergeyBiryukov So, would be some `assertStringContainsString` checks for those values?